articles/flows/guides/device-auth/includes/request-tokens.md

```diff
@@ -0,0 +1,85 @@
+## Request Tokens
+
+While you are waiting for the user to activate the device, begin polling the token URL to request an Access Token. Using the extracted polling interval (`interval`) from the previous step, you will need to `POST` to the [token URL](/api/authentication#device-auth) sending along the `device_code`.
+
+To avoid errors due to network latency, you should start counting each interval after receipt of the last polling request's response. 
+
+### Example request token POST to token URL
+
+```har
+{
+  "method": "POST",
+  "url": "https://${account.namespace}/oauth/token",
+  "headers": [
+    { "name": "Content-Type", "value": "application/x-www-form-urlencoded" }
+  ],
+  "postData": {
+    "text": "{\"grant_type\":\"urn%3Aietf%3Aparams%3Aoauth%3Agrant-type%3Adevice_code\",\"device_code\": \"YOUR_DEVICE_CODE\", \"client_id\": \"${account.clientId}\" }"
+  }
+}
+```
+
+#### Token Request Parameters
+
+| Parameter Name  | Description |
+|-----------------|-------------|
+| `grant_type`    | Set this to "urn:ietf:params:oauth:grant-type:device_code". This is an extension grant type (as defined by Section 4.5 of [RFC6749](https://tools.ietf.org/html/rfc6749#section-4.5)). |
+| `device_code`   | The `device_code` retrieved in the previous step of this tutorial. |
+| `client_id`     | Your application's Client ID. You can find this value in your [Application Settings](${manage_url}/#/Applications/${account.clientId}/settings). |
+
+### Token Responses
```

Comment:
This seems like a repeat of `articles/api/authentication/api-authz/_get-token.md`. Why do we have duplicate content here? This seems to be better formatted and explains things better.

Reply:
Same as my prior answer--this is part of a tutorial whereas the other is the API reference.

Reply:
But thank you. This doc is my writing/style, and the other isn't. :)